### PR TITLE
Cicero now allows Ergo modules

### PR DESCRIPTION
**Pull request by jeromesimeon**

### CODE REVIEW

File: packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json [hunk 1]
```diff
@@ -1 +1 @@
-{"$class":"org.accordproject.latedeliveryandpenalty.TemplateModel","clauseId":"2a997d12-b5c6-4ff8-b83e-ee72a0d73cae","forceMajeure":true,"penaltyDuration":{"$class":"org.accordproject.time.Duration","amount":9,"unit":"days"},"penaltyPercentage":7,"capPercentage":2,"termination":{"$class":"org.accordproject.time.Duration","amount":2,"unit":"weeks"},"fractionalPart":"days"}
\ No newline at end of file
+{"$class":"org.accordproject.latedeliveryandpenalty.TemplateModel","clauseId":"d1069bb3-8485-458f-ae6f-f9959efac8df","forceMajeure":true,"penaltyDuration":{"$class":"org.accordproject.time.Duration","amount":9,"unit":"days"},"penaltyPercentage":7,"capPercentage":2,"termination":{"$class":"org.accordproject.time.Duration","amount":2,"unit":"weeks"},"fractionalPart":"days"}
```
Comment (mttrbrts) [inline packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json]: Not blocking, but we should resolve the test that changes this ID on every run!

**Comment by jeromesimeon** [comment]
Fix for #245

File: packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json (hunk 1 above)
Comment (jeromesimeon) [reply packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json]: Yes please... Although, this is unrelated to this PR. I'll look at it.

Comment (jeromesimeon) [reply packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json]: This is a test for `cicero parse --out filename` is this something we want to remove from the CLI?

Comment (mttrbrts) [reply packages/cicero-cli/test/data/latedeliveryandpenalty/contract.json]: Let's keep the test, but add the file to `.gitignore`?

**Pull request merged**